docs/content/use_cases/items/question_the_docs.md

```diff
@@ -1,5 +1,13 @@
 # Ask the docs anything about SuperDuperDB
 
+In this notebook we show you how to implement the much-loved document Q&A task, using SuperDuperDB
+together with MongoDB.
+
+
+```python
+!pip install superduperdb==0.0.12
```

Comment:
Does version need to be hardcoded ? 

If yes, consider `>=` in order to be future-proof.

Reply:
Yes I can remove these since we are trying to guarantee support for these.